app/validation.go

```diff
@@ -79,6 +91,38 @@ func checkForUndefinedBOSHProperties(label string, properties map[string]string,
 	return allErrs
 }
 
+// checkParentsOfUndefined walks the chain of parents for `p` from the
+// bottom up and checks if any of them exist. The elements of the
+// chain are separated by dots.
+func checkParentsOfUndefined(p string, bosh propertyDefaults) bool {
+	at := strings.LastIndex(p, ".")
+
+	for at >= 0 {
+		// While there is a dot in the property name we have a
+		// parent to check the existence of
+
+		tail := p[at:]
+		parent := strings.TrimSuffix(p, tail)
```

Comment:
__Nothing__ done with commit 30fa129e639b70481b2c9c8595dd1b9e8d212fdd
(I really consider just using `p[at:]` to be a headache when having to work with the code later again).